### PR TITLE
feat(headless-styles): update Modal theme for Chakra - combine with C…

### PR DESCRIPTION
**Pull request by Rykus0**
…onfirmDialog

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

contributes to #487 

## What is the new behavior?

Replaces existing Modal theme provided by ConfirmDialog with one that works for both AlertDialog and Modal
Made slight adjustments to existing (ConfirmDialog) theme
Replaced with ChakraModal since the name more closely matches Chakra's base component theme name (Modal)

### CODE REVIEW

**Comment by Rykus0** [comment]
Temporary sandbox for comparing modal and alert dialog side-by-side:
https://codesandbox.io/s/ps-react-chakra-forked-9jy6ui?file=/package.json:506-507

Modal sandbox
https://codesandbox.io/s/basic-modal-react-chakra-zh2czl?file=/src/App.tsx

File: packages/headless-styles/src/components/Modal/chakraModal.ts [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
 const chakraAlertReset = {
-  paddingBottom: 0,
-  paddingInlineEnd: '1.5rem',
-  paddingInlineStart: '1.5rem',
-  paddingTop: 0,
+  padding: 0,
 }
 
-export const ChakraAlertDialog = {
+/* NOTE: AlertDialog also uses the Modal theme */
```
Comment (caseybaggz) [inline packages/headless-styles/src/components/Modal/chakraModal.ts:11]:
```suggestion
// NOTE: AlertDialog also uses the Modal theme
```

**Review by caseybaggz — COMMENTED** [review]
We need to also kill the chakraAlertDialog file if we are no longer using it.

**Comment by Rykus0** [comment]
> We need to also kill the chakraAlertDialog file if we are no longer using it.

Yeah, this is gone.  Git is showing it as renamed - it threw me, too!

**Review by caseybaggz — APPROVED** [review]

**Pull request merged**